src/algebra/lie/ideal_operations.lean

```diff
@@ -73,6 +73,9 @@ end
 lemma lie_mem_lie (x : I) (m : N) : ⁅(x : L), (m : M)⁆ ∈ ⁅I, N⁆ :=
 by { rw lie_ideal_oper_eq_span, apply subset_lie_span, use [x, m], }
 
+lemma lie_mem_lie' {x : L} {m : M} (hx : x ∈ I) (hm : m ∈ N) : ⁅x, m⁆ ∈ ⁅I, N⁆ :=
+N.lie_mem_lie I ⟨x, hx⟩ ⟨m, hm⟩
```

Comment:
This to me looks like it should be the unprimed name; the one above seems like saying `(S : add_submonoid R) (a b : S) : a + b ∈ S` which is really just the statement `↑x ∈ S` combined with the statement `↑(a + b)` = `↑a + ↑b`.

Is it true (or even stateable), that `(x : I) (m : N) : ⁅(x : L), (m : M)⁆ = ↑(⁅x, m⁆)`?

Reply:
I could be confused but I don't see the analogy with the `add_submonoid` example you give. Note that the proof of `lie_mem_lie` is not just `refl`.

With regard to your question, the following is true:
```lean
example (x : I) (m : N) : ⁅(x : L), (m : M)⁆ = ↑⁅x, m⁆ := rfl
```
but there's a bit more going on here than what is required for `lie_mem_lie` because this requires Lean to recognise that the `L`-module `N` can actually be regarded as an `I`-module which requires `lie_subalgebra.lie_ring_module` etc.

In any case I don't mind at all which of the two statements is called `lie_mem_lie` and `lie_mem_lie'` so if you have a strong feeling, just let me know and I'll adjust.

Reply:
I think this lemma could indeed be called `lie_mem_lie`, and the lemma above could be `lie_coe_mem_lie`.
@ocfnash do you think that makes sense?

Reply:
Thanks, this makes sense! Done.